price-feeder/config/config_test.go

```diff
@@ -133,13 +133,13 @@ func TestValidate(t *testing.T) {
 					RPCTimeout:    "100ms",
 				},
 				Telemetry: config.Telemetry{
-					ServiceName:         "price-feeder",
-					Enabled:             true,
-					EnableHostname:      true,
-					EnableHostnameLabel: true,
-					EnableServiceLabel:  true,
-					GlobalLabels:        make([][]string, 1),
-					Type:                "generic",
+					ServiceName:             "price-feeder",
+					Enabled:                 true,
+					EnableHostname:          true,
+					EnableHostnameLabel:     true,
+					EnableServiceLabel:      true,
+					GlobalLabels:            make([][]string, 1),
+					PrometheusRetentionTime: 120,
```

Comment:
To reduce the size of this test, you can introduce and reuse a `validTelemetry` struct, similar to how Robert did with [token_test.go](https://github.com/umee-network/umee/blob/main/x/leverage/types/token_test.go#L66) in leverage.

Reply:
yes, thanks for comment @toteki